config/secrets/local.ts

```diff
@@ -0,0 +1,9 @@
+const local = {
```

Comment:
Do we need this secrets config file? We can use the env-variables from env.ts (https://github.com/ost-cas-fee-adv-23-24/nextjs-app-football-fanatics/blob/main/config/env.ts)

Reply:
I will work on that a bit later. the idea is to set all variables in a config file and load the configs from there each time we need something. No Process.env in the code. only in one file.

Reply:
PR 11 contains a removal of the config